src/tests/ftest/datamover/copy_basics.py

```diff
@@ -23,7 +23,6 @@
 '''
 from data_mover_test_base import DataMoverTestBase
 from os.path import join, sep
-from apricot import skipForTicket
```

Comment:
@ashleypittman This import is need for @skipForTicket("DAOS-5512") below as well, so this file will error.